### DISCUSSION TITLE
Welcome to Alwatr Discussions!

### DISCUSSION THREAD

**Discussion by alimd** [AliMD/alwatr#54] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

درضمن ما زبان فارسی رو دوست داریم و هیچ مشکلی در فارسی نوشتن نداریم 💖

**Comment by alimd** [comment]
For the maintainers, here are some tips 💡 for getting started with Discussions.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.
  
  @MM25Zamanian

**Reply by mahdikarimi1** [reply]
Hello 
i am Mahdi Karimi and I am interested in learning programming.

**Comment by moodi2005** [comment]
hi

**Comment by amir25n** [comment]
Hello

**Comment by armanasadian** [comment]
Hi...!

**Comment by sahbaee** [comment]
:)

**Comment by MM25Zamanian** [comment]
Hi ;)

**Comment by berenjian** [comment]
Hi 😍

**Comment by amirali-rahmati** [comment]
hallo

**Comment by njfamirm** [comment]
👋

**Comment by MHHMAHDAVI** [comment]
Hi👋
I am Mohammad Hossein Mahdavi😎
I know html and css to some extent.
I worked a little on javascript. And I love typescript and backend.

**Comment by KianooshSoleimani** [comment]
Hi Mr MihanDoost
I am really in love with programming and enjoy it...
I know a little! about html, css, js, react, react-native, andoird-java ( a far time ago) 
Thank you ❤️

**Comment by reza-boroushaki** [comment]
Hi,
Would be an honour to join the community!

**Reply by alimd** [reply]
سلام رضا جان!
شما کجای دنیا هستی و چه میکنی؟
وبینار شرکت کرده بودی ولی اومدم زنگ بزنم شماره اشتباه بود گویا
تونستی واتساپ یک پیامی به من بده

**Comment by adltalab** [comment]
Hello 
I am interested in scientific Programming, Deep learning, and Challenging problems in various fields. 
I have Experienced in plasma simulation with C/C++ and little experience in AR/VR.
It would be an honor to join the community!

**Reply by adltalab** [reply]
https://github.com/adltalab/Cl_repo

**Comment by MohammadMostafaAR** [comment]
Hi!
my name is Mohammad Mostafa Akhlaghi Rezapour(it's a little long but you can call me Mostafa).
I'm almost 16 years old.
our joint friend Introduced you to me.
I've tried multiple Languages (of programming) but the only serious one was C#. 
Among the different branches of programming, I chose web programming.

**Comment by mmahdavis** [comment]
hi
I worked in different path: 
frontend   --> html, css, js, react
backend   --> php, django 
android   --> eclipse, b4a, android studio
design   --> photoshop, xd, figma, illustrator
game   --> game maker, unity
3d   --> sketchup, blender, autocad
dreamweaver, AI, ML, adobe premiere etc

but now i`m working with html5, tailwindcss, js and interested to threejs

**Comment by mahshidhooshmand** [comment]
Hi.
I've just started learning CSS,HTML and JavaScript . I'm just a beginner. I look forward to learning a lot from you🙏

**Comment by SMHosseini77** [comment]
👍

**Comment by shosseinfarjadi** [comment]
سلام حسینم در زمینه برنامه نویسی خیلی کم پایتون کار کردم و حوزه دیگری نرفتم سراغش حدودا از 4 سالگی پایه سیستم هستم و از سیستم عامل داس تا لینوکش و ویندوز xpاینارو همه رو دیدم در حوزه موشن گرافی انیمیشن سازی ادیتوری حرفه ای کار کردم و همیشه و هرروز اخبار گیم و مخصوصا تکنولوژی رو دنبال میکنم و تشنه یادگیری شدید هستم و به شکل عجیب و زیادی عاشق سخت افزار و گیم هستم
با زبان انگلیسی هم مشکل ندارم😅
Hello Hossein, I worked very little in the field of Python programming and I did not go to any other field. Every day I follow game news and especially technology and I am very thirsty for learning and I love hardware and games in a strange way.

**Comment by satafti** [comment]
سید علیرضا تفتی هستم 
بعد از ۹ سال دوباره میخوام برگردم وبرنامه نویسی یاد بگیرم

**Reply by alimd** [reply]
سلام
ریپوت کو؟

**Comment by amir91179** [comment]
سلام استاد وقتتون بخیر!
https://github.com/amir91179/First-Respository

**Comment by shosseinfarjadi** [comment]
https://github.com/shosseinfarjadi/title
سلام بفرمایید

**Comment by alirezaFAkhar** [comment]
سلام استاد خسته نباشید 
https://github.com/alirezaFAkhar/Travelo/edit/main/README.md

**Comment by ghost** [comment]
hello
my name is Mohammed Kazem Moghadasan Nishabory

https://github.com/MNKazem/caspian

**Comment by AbolfazlRasoulizade** [comment]
Hi I am Abolfazl Raoulizade.
I am at the beginning of programming!

https://github.com/abolfazlrasoulizade/Introduce-Myself/

**Comment by MuhammadMGH** [comment]
Hello teacher how are you ?
I'm Muhammad Ghafoorian and this is my project .
https://github.com/MuhammadMGH/My_first_repo/blob/main/README.md

**Comment by mahshidhooshmand** [comment]
Hello, here is my first project
https://github.com/mahshidhooshmand/my-first-repo

**Comment by sabermanesh** [comment]
سلام
امینم
کاملا مبتدی
https://github.com/sabermanesh/AminMD/blob/8ba2126d5497b41fbba3b54dd3d2898f89648c18/README.md

**Comment by A-sina** [comment]
salam va dorood
https://github.com/A-sina/thefirst/blob/main/README.md

**Comment by mahyasbm** [comment]
سلام
من محیا صابرمنشم، با تمام غریبی با این فضاها، تصمیم به یادگیری گرفتم.

https://github.com/mahyasbm/mohajer/blob/e90dd727ff21cd0e67932c34c8dab45eae095b0e/README.md

**Comment by amirmmolazemi** [comment]
Salam o dorood
 AmirMohammad Molazemi hastam 
 https://github.com/amir341mohamad/myfirstproject.git

**Comment by Rahimpanah** [comment]
Hello teacher!
this is my exercise about Markdown language.
https://github.com/Rahimpanah/my_first_pro/blob/main/README.md

thanks for your support,
Reyhaneh

**Comment by M-Beheshti** [comment]
سلام محمد بهشتی هستم 
خوشحالم ک بلاخره شروع کردم این راه تموم نشدنی رو :)
https://github.com/M-Beheshti/My-First.git

**Comment by marihooshmand** [comment]
سلام ماریه هوشمند هستم
 چندان آشنایتی با این فضا ندارم ، دوست دارم که یاد بگیرم
https://github.com/marihooshmand/my-first-repo

**Comment by sepehrislami** [comment]
سلام

**Comment by Masterchoir** [comment]
سلام
https://github.com/Masterchoir/My-First-Repo-1

**Comment by awrashsajjadi** [comment]
باشه برای یادگاری 💎

**Comment by amiresp** [comment]
Hello @AliMD ;
I'm Amir Spalani and love web technology.
for now i use lit and I hope to learn from you next to you (same teammate)

**Reply by amiresp** [reply]
https://github.com/vaadin/web-components/issues/3622 
:trollface:

**Comment by MuhammadMGH** [comment]
Hello teacher
This is my text and in the text I used all the commands in this site (W3schools)
https://jsbin.com/bijarip/3

**Reply by MuhammadMGH** [reply]
please checking this link teacher
<h1 style="font-size:50px">Thanks</h1>

**Comment by MuhammadMGH** [comment]
https://github.com/MuhammadMGH/MuhammadMGH.github.io

**Comment by awrashsajjadi** [comment]
سلام و درود 
https://github.com/awrashsajjadi/my-first

**Comment by alirezaFAkhar** [comment]
سلام استاد 
علیرضا فخار هستم خوش حالم که تو کلاس شما شرکت کردم
https://github.com/alirezaFAkhar

**Comment by mohamadhooseinHR** [comment]
سلام و خسته نباشید خدمت استاد میهن دوست و بقیه دانش اموزان عزیز 
محمد حسین هستم  و امیدوارم در مسیری ک ادامه میدم هیچ چیزی مانع راهم نشه 
https://github.com/mohamadhooseinHR/my-first-experience

**Comment by alibqn** [comment]
سلام استاد وقتتون بخیر سید علی باقریان هستم باعث افتخار هستش که در دورتون شرکت میکنم 
https://github.com/AliBqn
https://github.com/ProFoSSoR/My-First-Repo

**Comment by KianooshSoleimani** [comment]
I know this is very late but It is never too late to mend 😉

This site   [My First Practice](https://github.com/KianooshSoleimani/markdown_practice).

**Comment by sepehrislami** [comment]
[my first repository](//github.com/sepehrislami/my-first-repository)

**Comment by MuhammadMGH** [comment]
<!DOCTYPE html>
<html>
<head>
</head>
<body>
  <a class="indent" href="https://output.jsbin.com/renivox">This is my first home page</a>
</body>
</html>

**Comment by Arian-Akhavan** [comment]
سلام و درود     
  https://github.com/Arian-Akhavan/my-first-repo

**Comment by MuhammadMGH** [comment]
<h1> Salam Ostad </h1>
<p> Baraye didan proje man roye<a href="https://MuhammadMGH.github.io/README.md"> here </a>click konid </p>
thanks!

**Reply by MuhammadMGH** [reply]
<h1>please Check my project</h1>

**Comment by Ostad-Tavana** [comment]
Hi I am Erfan and I newcomer.

**Reply by shosseinfarjadi** [reply]
سلام عرفان خوش اومدی🥳

**Comment by Mohsen-Haghighi** [comment]
mohsen haghighi. salam be hame dustan.

**Comment by MohammadHoseynSeyedi85** [comment]
Hi! I'm Mohammad Hoseyn Seyedi .

**Comment by MohammadHoseynSeyedi85** [comment]
This is my homework about third session.
I tried to pay attention to the points that you taught and some points on the GitHub site. thanks ...

https://github.com/MohammadHoseynSeyedi85/About_me-/blob/main/README.md

**Comment by FatemeGharavian** [comment]
salam ostad 

https://github.com/FatemeGharavian/my-frist-repol

**Comment by lordaghasi** [comment]
hello dear teacher. my name is Mohamad Aghasi. i want to learn about css and javascript. you know I am really noob :D

**Comment by MohammadHoseynSeyedi85** [comment]
استاد سلام . خسته نباشید .
یک خواهشی داشتم ازتون . 
ایا امکانش هست یک وبینار بزارید و در رابطه با پروژه الوتر و نحو بهره گیری از این پروژه توضیحاتی به ما بدهید؟ 
چون به شخصه خیلی دلم میسوزه که یادندارم از این همه کد و .... و اموزش هایی که در دل اینها نهفته هست استفاده بکنم .

**Comment by kazemi2006** [comment]
hi my name is akbar kazemi
I'm 16

**Reply by kazemi2006** [reply]
https://github.com/kazemi2006/my-project#my-project

**Reply by alimd** [reply]
سلام
پیامت رو الان دیدم!
🌹

**Comment by SMsajjadSM** [comment]
Hi guys
 My name is seyed sajjad

I worked  html css front
Jquery js a bit
C# for windows
And now i want have new experience with Mr mihandost
Tnx

**Comment by Amirghda** [comment]
hy mr.mihandost.
nice to meet you ...
According to your advice, I created an account on GitHub.
I'm happy you follow me

**Reply by alimd** [reply]
سلام
خیلی خوش آمدی
۲ نکته
اول اینکه usernameای انتخاب کن که تا سالها تغییرش ندی
میتونی اینو الان تغییر بدی و یا نگه داری ولی دقیق بررسی کن که ترجیحا با مجموعه سایت های دیگه یکی باشه
دوم اینکه نام خودت رو روی یوزر بزار تا بایک نگاه مشخص بشه (به اکانت من دقت کن)
البته در بیو نوشتی ولی حرفه ای تر اینه که در بیو فقط توضیحات فنی باشه ولی بازم میل خودته

**Comment by SMsajjadSM** [comment]
first try with mrkdown
https://github.com/SMsajjadSM/sajjadrepository
tnx

**Comment by mousavimina** [comment]
سلام
مینا هستم. 
خوشحالم که بعد از حدود 12 سال دوری از دنیای برنامه نویسی، همراه شما و با کمک شما به این دنیای شیرین برمی گردم.
امیدوارم بتونم هرچه سریعتر خودم رو به کلاس برسونم.

**Reply by alimd** [reply]
سلام
خوش آمدید

**Reply by mousavimina** [reply]
Hi
mousavimina.github.io
This is my project link at github....

**Reply by mousavimina** [reply]
https://github.com/mousavimina/mousavimina.github.io

**Comment by AmirHosseinAbbasii** [comment]
سلام استاد وقت بخیر من اولین قدم رو انجام دادم و اکانت گیت هاب رو بالا اوردم امیدوارم در این راه ثابت قدم بمونم
https://github.com/AmirHosseinAbbasii/TheFirst.md پروژه markdown

**Comment by AmirrezaTN** [comment]
Hi guys 
My name is Amirreza Taghavinia

I'm 20 years old and I'm computer engineering student of IAUM
I'm worked with C++ just about university projects not professional 
I started working with computer when I was 4yo and I love working with computer especially playing video games😁

**Comment by AliPeivandi** [comment]
Hi there. Ali Peivandi...in the procecess of learning!

**Comment by AlirezaKhodayari** [comment]
Hello Mr.Mihandoost
The First Mission was done.
you can see it in this link:
https://github.com/AlirezaKhodayari/AirP1
Thanks a lot.
Have a good Time.
